types/jsonstream-next/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {

```

Comment:
Is it possible to disable these rules for specific lines of code and not the whole package?

Reply:
Since it's 3 functions and they all violate the lint rules, I think it's fine as-is. I removed all the other linting exceptions.